pages/docs/deployment/migrate-v2-to-v3.mdx

```diff
@@ -0,0 +1,223 @@
+---
+description: A guide to upgrade a Langfuse V2 setup to V3.
+---
+
+import { Callout } from "nextra/components";
+
+# Migrate Langfuse V2 to V3
+
+<Callout type="warning">
+  This guide covers a developer preview which is **not suitable for production use**.
+  V3 is under active development and we plan to ship a production-ready version by the end of November 2024.
+  We share this information to gather feedback from our awesome developer community.
+
+  For a production-ready setup, follow the [self-hosting guide](/docs/deployment/self-host)
+  or consider using [Langfuse Cloud](https://cloud.langfuse.com) maintained by the Langfuse team.
+
+  If you want to get started on V3, please follow our [V3 Self-Host Guide](/docs/deployment/self-host-v3).
+</Callout>
+
+## Reasoning
```

Comment:
I would probably move reasoning to the bottom and just refer to it with a link to the anchor at the top. Not relevant for "migrate" task but good to know as it motivates this change